bl-kernel/admin/views/settings.php

```diff
@@ -1,6 +1,6 @@
 <?php defined('BLUDIT') or die('Bludit CMS.'); ?>
 
-<?php echo Bootstrap::formOpen(array('id'=>'jsform', 'class'=>'tab-content')); ?>
+<?php echo Bootstrap::formOpen(array('id'=>'jsform', 'class'=>'tab-content', 'action'=>'settings.php')); ?>
```

Comment:
I feel action needs to be just `settings` instead of the file.

Reply:
No, in this case it must be `settings.php`, otherwise it wouldn't pass through the mod security filter:

```
<LocationMatch /admin/settings.php>
SecRuleRemoveById 340162
SecRuleRemoveById 340163
</LocationMatch>
```

The filter above removes the WAF Rule 340162, which prevents URLs from being passed / processed via a _POST request. We found the solution [here](https://github.com/bludit/bludit/issues/1216#issuecomment-650511142).